release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -568,6 +568,18 @@ The following enhancements to MetalLB and the MetalLB Operator are included in t
 
 For more information, see xref:../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
 
+
+[id="ocp-4-10-host-firmware-settings"]
+==== Support for modifying host firmware settings
+
+{product-title} supports the `HostFirmwareSettings` resource and the `FirmwareSchema` resource. When deploying OpenShift Container Platform on bare metal hosts, there are times when you need to make changes to the host either before or after provisioning. This can include inspecting the host's firmware and BIOS details. There are two new resources that you can use with the Bare Metal Operator (BMO):
```

Comment:
Suggestion: 
{product-title} supports the `HostFirmwareSettings` and `FirmwareSchema` resources.